src/components/organisms/AddForm/AddForm.jsx

```diff
@@ -0,0 +1,71 @@
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import PropTypes from 'prop-types';
+import { Icon } from '../..';
+
+import './Form.scss';
+
+export const AddForm = ({ placeholder, type }) => {
+    const submitOnEnter = (event) => {
+        if (event.key === 'Enter' && event.shiftKey === false) {
+            event.preventDefault();
+            event.target.form.dispatchEvent(
+                new Event('submit', { cancelable: true, bubbles: true })
+            );
+        }
+    };
+
+    const onSubmit = (values, action) => {
+        // TODO temporary, connect with API
+        action.resetForm({
+            values: {
+                [`${type}`]: ''
+            }
+        });
+    };
+
+    const focusOnInput = (event) => {
+        if (!event.target) {
+            event.target[0].focus();
+        }
+    };
+
+    const validation = Yup.object().shape({
+        [`${type}`]: Yup.string().required('Required')
+    });
+
+    return (
+        <div
+            className={`form form__${type}`}
+            role='directory'
```

Comment:
what is this prop doing? really, I don't know xd

Reply:
role is required to give onClick on div, but i don't know if values is suitable in this case, I picked this beacause this one works tbh XD